docs/advanced-topics/webhook.asciidoc

```diff
@@ -0,0 +1,19 @@
+:page_id: webhook
+ifdef::env-github[]
+****
+link:https://www.elastic.co/guide/en/cloud-on-k8s/master/k8s-{page_id}.html[View this document on the Elastic website]
+****
+endif::[]
+
+[id="{p}-{page_id}"]
+= Webhook namespace selectors 
+
+Users are now allowed to set `namespaceSelector` and `objectSelector` on the webhook. The webhook name is generated as ``<operator-name>.<operator-namespace>.k8s.elastic.co` so that multiple operators can be installed side-by-side in the same cluster. 
+
+This can be useful in large and busy clusters, where you might want to divide the set of namespaces across several operators to speed up reconciliation times and reduce the amount of resources required per operator.
+
+Webhook resources are cluster-scoped, therefore `createClusterScopedResources` must be true. This means that each operator gets a ClusterRole that is not restricted to the set of managed namespaces. This approach avoids over-complicating the chart and is suitable for load-splitting scenarios. 
+
+WARNING: It is not recommended to deploy webhook resources in environments where operators are run by untrusted users and need to be locked down tightly.
+
+For more information on webhook, see link:https://kubernetes.io/docs/reference/access-authn-authz/extensible-admission-controllers/[Dynamic Admission Control].
```

Comment:
Maybe point to https://www.elastic.co/guide/en/cloud-on-k8s/current/k8s-webhook.html as well?